src/tic_tac_toe/setup.clj

```diff
@@ -1,6 +1,7 @@
 (ns tic-tac-toe.setup
   (:use [clojure.java.io :as fs])
-  (:require [tic-tac-toe.board :as board]
+  (:require [tic-tac-toe.announcer :refer :all]
```

Comment:
Be careful of wildcard imports (or `:all` in Clojure). I will almost always avoid them except for a few very specific cases.

Your imports can give you a good idea of how many dependencies your module has, and give you clues when it might have too many responsibilities. This is why I tend to avoid it.

Reply:
[Adjusted](https://github.com/saralein/tic-tac-toe_clojure/pull/10/files#diff-4fbcd66ffb99e118519711e232783759R4) this a bit so messages can be passed around with other information.  This way, other namespaces don't use announcer (which did feel weird).